.gitignore

```diff
@@ -5,4 +5,6 @@ node_modules
 *.tmp
 .vs
 dev.targets
```

Comment:
Is that not getting used anymore?

Can you also update readme as well please

Reply:
done :+1: